java/src/main/native/CMakeLists.txt

```diff
@@ -146,11 +151,13 @@ endif (CUDF_INCLUDE AND CUDF_LIBRARY)
 
 find_path(RMM_INCLUDE "rmm"
           HINTS "$ENV{RMM_ROOT}/include"
```

Comment:
Is this necessary? If others are OK I would like to remove this

Reply:
This allows users to override the prefix where RMM has been installed via the `RMM_ROOT` environment variable, just as they can with cudf via `CUDF_ROOT`.